plugins/shorten_urls.lua

```diff
@@ -74,8 +75,10 @@ return {
 					end
 				end
 				if #msg > 0 then
-					msg = sender[1] .. ": " .. msg
-					irc:PRIVMSG(origin, msg)
+					no_width_space = utf8.char(8203)
```

Comment:
Should be `local`. However you can just write it inside a string (either literally or via `"%1\u{200b}%2"`)